.ci/scripts/distribution/it-java.sh

```diff
@@ -1,13 +1,19 @@
 #!/bin/bash -eux
 
-
-export JAVA_TOOL_OPTIONS="$JAVA_TOOL_OPTIONS -XX:MaxRAMFraction=$((LIMITS_CPU))"
-
+LIMITS_CPU=${LIMITS_CPU:-$(getconf _NPROCESSORS_ONLN)}
+FORK_COUNT=${FORK_COUNT:-}
```

Comment:
Perhaps we should just default this explicitly to 1

Reply:
That's different behaviour though. Right now, this is set up so you can override whatever's defined in the POM - if we explicitly set to 1, then we __always__ override it, which I don't know if we really want to do. I think ignoring it if unset is a little more intuitive in my opinion.